File: solutions/go/darts/2/darts.go
package darts

import "math"

func Score(x, y float64) int {
	radius := math.Hypot(x, y)
    switch {
    case radius > 10:
        return 0
    case radius > 5:
        return 1
    case radius > 1:
        return 5
    default:
        return 10
    }
}

File: solutions/java/darts/2/src/main/java/Darts.java
class Darts {
    int score(double xOfDart, double yOfDart) {
        final var radius = Math.hypot(xOfDart, yOfDart);
        return (radius > 10) ? 0 :
               (radius > 5) ? 1 :
               (radius > 1) ? 5 : 10;
    }
}
